Tried fixing problem #20

arv-77 opened the pull request:
I have tried solving the #20 problem by reducing the font-size on decreasing the width of the webpage so that the header fits in the same line.

arv-77 committed:
Tried fixing problem #20

arv-77 committed:
Tried to make POTW responsive #25

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
# This is a combination of 4 commits.
# This is the 1st commit message:

# This is a combination of 2 commits.
# This is the 1st commit message:

# This is a combination of 7 commits.
# This is the 1st commit message:

# This is a combination of 17 commits.
# This is the 1st commit message:

Testing #20

# This is the commit message #2:

Testing #20

# This is the commit message #3:

Testing #20

# This is the commit message #4:

Testing #20

# This is the commit message #5:

Testing #20

# This is the commit message #6:

Testing #20

# This is the commit message #7:

Testing #20

# This is the commit message #8:

Testing #20

# This is the commit message #9:

Testing #20

# This is the commit message #10:

Testing #20

# This is the commit message #11:

Testing #20

# This is the commit message #12:

Testing #20

# This is the commit message #13:

Testing #20

# This is the commit message #14:

Testing #20

# This is the commit message #15:

Testing #20

# This is the commit message #16:

Testing #20

# This is the commit message #17:

Tried to make POTW responsive #25

# This is the commit message #2:

Testing #20

# This is the commit message #3:

Testing #20

# This is the commit message #4:

Testing #20

# This is the commit message #5:

Testing #20

# This is the commit message #6:

Testing #20

# This is the commit message #7:

Testing #20

# This is the commit message #2:

Testing #20

# This is the commit message #2:

Testing #20

# This is the commit message #3:

Testing #20

# This is the commit message #4:

Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

arv-77 committed:
Testing #20

shivanshs9 reviewed (changes requested):
Sorry, but that's not how commits are done. Clean up your commits, by squashing them to one.

arv-77 committed:
# This is a combination of 8 commits.
# This is the 1st commit message:

Merge branch 'master' of https://github.com/arvindmuralie77/cops-website

# This is the commit message #2:

Testing #20

# This is the commit message #3:

Testing #20

# This is the commit message #4:

Testing #20

# This is the commit message #5:

Testing #20

# This is the commit message #6:

Testing #20

# This is the commit message #7:

Testing #20

# This is the commit message #8:

Testing #20

arv-77 committed:
Merge branch 'master' of https://github.com/arvindmuralie77/cops-website

arv-77 committed:
Merge branch 'master' of https://github.com/arvindmuralie77/cops-website

arv-77 committed:
#20

shivanshs9 commented:
Sorry, but it seems the number of your commits are rising exponentially. I'll have to close this PR if you still aren't able to clean it up. I would suggest you to practice your git skills with https://learngitbranching.js.org/
